admin/javascript/src/components/form-action/index.js

```diff
@@ -100,7 +101,7 @@ class FormActionComponent extends SilverStripeComponent {
 
 FormActionComponent.propTypes = {
   id: React.PropTypes.string,
-  handleClick: React.PropTypes.func.isRequired,
+  handleClick: React.PropTypes.func,
```

Comment:
If `handleClick` is no longer required we should put a guard in `this.handleClick` so that `this.props.handleClick` isn't invoked when it's `undefined`